app/main.ipynb

```diff
@@ -192,18 +239,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "    \n",
+    "mean_value = A.mean(axis=1)\n",
+    "print(mean_value)"
```

Comment:
The mean is calculated along the wrong axis. You should calculate the mean along the columns (axis=0) to find the mean for each column as required by the task.

Reply:
but task is Find the mean for each column by `x` axis